include/string.h

```diff
@@ -35,7 +35,9 @@
  ****************************************************************************/
 
 #define strcoll_l(s1, s2, l)    strcoll(s1, s2)
+#define strdupa(x)              strcpy(alloca(strlen(x) + 1), x)
 #define strerror_l(e, l)        strerror(e)
+#define strndupa(x, len)        strncpy(alloca(strlen(x) + 1), x, len)
```

Comment:
The implementation looks wrong:
https://github.com/esmil/musl/blob/master/include/string.h#L89-L95

Reply:
Done!